package.json

```diff
@@ -2,26 +2,38 @@
   "name": "sgid-test-app",
   "version": "2.0.0",
   "description": "This is an example node application that implements sgID's OAuth2 API.",
-  "main": "index.js",
+  "main": "dist/index.js",
   "engines": {
     "node": ">=16.16"
   },
   "dependencies": {
-    "@opengovsg/sgid-client": "^1.0.4",
+    "@opengovsg/sgid-client": "github:opengovsg/sgid-client#test/v2",
+    "@types/uuid": "^9.0.1",
+    "cookie-parser": "^1.4.6",
     "dotenv": "8.6.0",
     "ejs": "3.1.7",
     "express": "4.17.1",
     "helmet": "4.6.0",
     "jsonwebtoken": "8.5.1",
+    "node-cache": "^5.1.2",
     "node-fetch": "2.6.7",
-    "node-jose": "2.1.1"
+    "node-jose": "2.1.1",
+    "ts-node": "^10.9.1",
```

Comment:
should be a dev dep?

Reply:
Resolved in [6a21f7a](https://github.com/opengovsg/sgid-test-app/pull/94/commits/6a21f7abfb16f44d9ca5bc9fcf4c6c4e26fd6910).